README.md

```diff
@@ -0,0 +1,129 @@
+# Project Nebula - Development README
+
+## Welcome to the Project Nebula Development Repository!
+
+This repository houses the codebase and assets for **Project Nebula**, a game currently in development for [The Escape 2024 Game Jam](https://itch.io/jam/the-2024-escape-jam). Below, you'll find guidelines and resources for contributing to the project, setting up your development environment, and understanding the structure of the codebase.
+The target platform for this project is the web browser, and thus it will be built for `WebGL`.
+
+## Table of Contents
+1. [Getting Started](#getting-started)
+2. [Project Structure](#project-structure)
+3. [Development Guidelines](#development-guidelines)
+4. [Branching Strategy](#branching-strategy)
+5. [Coding Standards](#coding-standards)
+6. [Assets and Resources](#assets-and-resources)
+7. [Testing](#testing)
+8. [Issue Tracking](#issue-tracking)
+9. [Continuous Integration](#continuous-integration)
+
+## Getting Started
+
+### Prerequisites
+- **Unity Version:** Make sure you have the `2023.2.^` version of Unity installed. The project uses the **Universal Render Pipeline (URP)** with 2D Renderer, so ensure your Unity setup supports it.
+- **Version Control:** We use Git for version control. Ensure you have Git installed and are familiar with basic commands.
+
+### Cloning the Repository
+To get started, clone the repository to your local machine:
+
+```bash
+git clone https://github.com/flowmotion-bbd/game-jam-2024-bbd.git
+cd game-jam-2024-bbd
+```
+
+### Setting Up Unity
+1. Open Unity Hub.
+2. Click "Add" and navigate to the directory where you cloned the repository.
+3. Select the project and open it.
+
+### Installing Dependencies
+Upon opening the project, Unity will automatically resolve any package dependencies via the Unity Package Manager. Ensure all packages are installed without errors.
+
+## Project Structure
```

Comment:
I think this project structure might needs some work

Reply:
What does ye suggest?